### DISCUSSION TITLE
How to access local file system ?

### DISCUSSION THREAD

**Discussion by captain-majid** [rejetto/hfs#809] [Q&A]
- I'm using this to allow access to local files:
`Chrome --allow-file-access-from-files file:\\\E:\shared\starcraft\bwapi-data\BR\OpenBW-ReplayViewer\index.html?rep="E:\shared\starcraft\maps\replays\! PvT Bisu,Hiya@F-Spirit.rep"`

Problem is I've to close all Chrome instances before starting with this switch, and it seems not safe/wise at all to allow all sites to access my file system, just to make 1 already local file (.html) to access another local file (.rep). I've heard local HTTP servers can do this easily, any idea how to do that with HFS ?

**Comment by rejetto** [comment]
that's basically what hfs is for, to expose some files or some folders, you decide.
you select the files from the "shared files" page.
i'm moving this to "discussions".

**Comment by cbr00t** [comment]
> Problem is I've to close all Chrome instances before starting with this switch

Starting chrome or msedge with
   chrome.exe   **--user-data-dir=c:\path\to\tempProfile**
   msedge.exe  **--user-data-dir=c:\path\to\tempProfile**
arg launches independent instance of browser so new args can be used without closing other browser instances.
in short: launches with a seperate user profile browser window.

https://www.chromium.org/developers/creating-and-using-profiles/#to-use-the-profile
